Package-Config.roc

```diff
@@ -11,10 +11,13 @@ initForHost = init
 # TODO: once `roc glue` knows how to resolve them, these should move back
 # into Rbt.roc so we can stop copying the definitions over every time we make
 # a change!
-Tool : [SystemTool { name : Str }]
+SystemToolPayload : { name : Str }
+Tool : [SystemTool SystemToolPayload]
 
-Command : [Command { tool : Tool, args : List Str }]
+CommandPayload : { tool : Tool, args : List Str }
+Command : [Command CommandPayload]
 
-Job : [Job { command : Command, inputFiles : List Str, outputs : List Str }]
+JobPayload : { command : Command, inputFiles : List Str, outputs : List Str }
```

Comment:
Do we want to do the same for Job and Command?

Make them just directly define to the inner struct instead of a union?

Reply:
I don't think we'll be able to. I know I'm gonna have to move at least some of these back inside the unions once Job has Job dependencies (a Dict Str Job eventually). Same with Tool, which will grow "FromJob { name : Str, job : Job }"